cilium/cmd/fqdn.go

```diff
@@ -43,31 +50,83 @@ var fqdnCleanCacheCmd = &cobra.Command{
 	},
 }
 
-var fqdnCacheClearMatchPattern string
+var fqdnListCacheCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List fqdn cache contents",
+	Run: func(cmd *cobra.Command, args []string) {
+		listFQDNCache()
+	},
+}
+
+var fqdnCacheMatchPattern string
 
 func init() {
+	fqdnCacheCmd.AddCommand(fqdnListCacheCmd)
 	fqdnCacheCmd.AddCommand(fqdnCleanCacheCmd)
 	fqdnCmd.AddCommand(fqdnCacheCmd)
 	rootCmd.AddCommand(fqdnCmd)
 
 	fqdnCleanCacheCmd.Flags().BoolVarP(&force, "force", "f", false, "Skip confirmation")
-	fqdnCleanCacheCmd.Flags().StringVarP(&fqdnCacheClearMatchPattern, "pattern", "p", "", "Cache entries with fqdn matching the pattern will be deleted")
+	fqdnCleanCacheCmd.Flags().StringVarP(&fqdnCacheMatchPattern, "pattern", "p", "", "Cache entries with fqdn matching the pattern will be deleted")
+
+	fqdnListCacheCmd.Flags().StringVarP(&fqdnCacheMatchPattern, "pattern", "p", "", "List cache entries with FQDN matching the pattern")
```

Comment:
I just realised that this is more consistent with our policies if we call it `matchpattern`. Also, we use a capitalised FQDNs here, but lowercase with the delete option. I would use FQDNs.
Separately, the two help texts are a little different. I'd prefer leading with the verb, since it is then clear what is about to happen:
`Delete cache entries with FQDNs that match matchpattern`
`List cache entries with FQDNs that match matchpattern`

Reply:
done